www/nginx/src/opnsense/mvc/app/views/OPNsense/Nginx/ban.volt

```diff
@@ -38,6 +38,7 @@
         </tbody>
     </table>
 </div>
+<script src="{{ cache_safe('/ui/js/moment-with-locales.min.js') }}"></script>
```

Comment:
moment should not be needed anymore. Browsers should nowadays have a proper time display and formatting api. 
Same for the other inclusion:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
Hi! thanks for review! yep, i think i used `moment` because of the reverse local-to-epoch time conversion in bootgrid converter. since it turned out that Bans grid never triggers the `from` conversion, i agree that `moment.js` is not required. so may be `toLocaleString` is enough?